search-service/src/main/resources/application.properties

```diff
@@ -13,3 +13,6 @@ server.port = 8083
 
 # Allow body size in payloads up to 2Mb
 search.payload-max-body-size = 2048000
+
+# Allow creator of an entity to delete it
+search.delete-entity-by-creator = false
```

Comment:
Is there a name that explain what the variable do?
for example : 
- delete-entity-with-their-owner
- owner-cascade-delete
- other?

That way you can get rid off the comment to.

Reply:
- `owner-delete-cascade-entities`
- `on-owner-delete-cascade-entities`